src/main/java/org/gridsuite/explore/server/ExploreController.java

```diff
@@ -300,23 +272,21 @@ public ResponseEntity<Void> createParameters(@RequestBody String parameters,
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "parameters have been successfully modified")})
     public ResponseEntity<Void> updateParameters(@PathVariable UUID id,
                                              @RequestBody String parameters,
-                                             @RequestParam(name = QUERY_PARAM_PARAMETERS_TYPE, defaultValue = "") ParametersType parametersType,
+                                             @RequestParam(name = QUERY_PARAM_TYPE, defaultValue = "") ParametersType parametersType,
                                              @RequestHeader("userId") String userId,
                                              @RequestParam("name") String name) {
         exploreService.updateParameters(id, parameters, parametersType, userId, name);
         return ResponseEntity.ok().build();
     }
 
-    @PostMapping(value = "/explore/parameters")
+    @PostMapping(value = "/explore/parameters", params = "duplicateFrom")
     @Operation(summary = "Duplicate parameters")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "parameters have been successfully duplicated")})
-    public ResponseEntity<Void> duplicateParameters(@RequestParam("duplicateFrom") UUID parentParameterId,
-                                                 @RequestParam("name") String parametersName,
-                                                 @RequestParam(name = QUERY_PARAM_PARAMETERS_TYPE, defaultValue = "") ParametersType parametersType,
-                                                 @RequestParam(name = "description", required = false, defaultValue = "") String description,
-                                                 @RequestParam(QUERY_PARAM_PARENT_DIRECTORY_ID) UUID parentDirectoryUuid,
-                                                 @RequestHeader("userId") String userId) {
-        exploreService.duplicateParameters(parentParameterId, parametersType, parametersName, description, parentDirectoryUuid, userId);
+    public ResponseEntity<Void> duplicateParameters(@RequestParam("duplicateFrom") UUID parametersId,
+                                                    @RequestParam(name = QUERY_PARAM_PARENT_DIRECTORY_ID, required = false) UUID targetDirectoryId,
+                                                    @RequestParam(name = QUERY_PARAM_TYPE, defaultValue = "") ParametersType parametersType,
```

Comment:
why an empty defaultValue ?
> Supplying a default value implicitly sets required to false
Which service is called then ? is there a TU to test this ?

Reply:
done